base_app/src/app/cmdsets/material-popup-component.html

```diff
@@ -34,13 +32,13 @@
 
             <mat-form-field appearance="fill" *ngIf='control.type === "bool"'>
                 <input matInput type='checkbox' [id]="'popup-control-' + control.id"
-                       [formControl]="inputControl" 
+                       [formControl]="inputControl"
                        [attr.aria-label]="'labelFor-' + control.id"
-                       [checked]="control.checked"  
+                       [checked]="control.checked"
                        [name]="control.name"
                        [placeholder]="control.name"
                        [title]="control.desc">{{control.label}}
-                />
```

Comment:
looks like you lost your closing tag

Reply:
I thought the same thing too, but its an input tag so it's format is `<input [name]=... >`

Reply:
you definitely still need it, also you don't have a `>` at all as far as I can see

Reply:
it's still there, it's hidden inbetween `[title]="control.desc"` and `{{control.label}}`

Reply:
Ah okay, i wonder why that /> was ever there then. I'd put the {{control.label}} on another line since it's a child of the <mat-form-field> element, not the <input> element

Reply:
I think it might've just been a typo or confusion with another js framework. I think with JSX you need `<input ..... />` which is similar. I noticed the other input tags in the file are set up as the JSX format which confused me so I went to [angular's documentation](https://angular.io/guide/forms) and found an example where they use `<input ......>`. I changed the rest of the file to fit this format and moved `{{control.label}}` to it's own line

Reply:
/> is definitely my error.  Surprised this worked at all.

Reply:
Yeah turns out that the "/>" is a JSX (XHTML) thing, in regular HTML it's almost always optional to have the "/"